app/views/posts/index.html.erb

```diff
@@ -1 +1,36 @@
-<h1>Listing User Posts</h1>
\ No newline at end of file
+<div class="user">
+  <div class="user-photo">
+    <img src="<%= @user.photo %>" alt="User Photo">
+  </div>
+  <div class="user-details">
+    <%= link_to user_url(@user), class: 'name' do %>
+      <%= @user.name %>
+    <% end %>
+    <p class="posts-no">
+      <%= "Number of posts: #{@user.posts_counter}" %>
+    </p>
+  </div>
+</div>
+<% @posts.each_with_index do |post, index| %>
+     <div class = "post">
+        <%= link_to user_post_url(user_id: @user.id, id: post.id), class: 'user-link' do %>
+          <h3>Post #<%= index + 1 %></h3>
+        <% end %>
+        <p><%= post.text %></p>
+          <div class="count">
+            <p>Comments: <%= post.comments_counter %></p>
+            <p>Likes: <%= post.likes_counter %></p>
+          </div>
+     </div>
+     <% if post.comments_counter > 0 %>
+      <ul class="comments">
+            <% post.comments.each do |comment| %>
+                <li><%= comment.author.name %>: <%= comment.text %></li>
+            <% end %>
+      </ul>
+     <%end%>
+  <% end %>
+  <div class="btn-link">
+    <button>Pagination</button>
+  </div>
```

Comment:
- [ ] Kindly make sure the views of the app look like the provided wireframe

|Your website|Provided Wirframes|
|-|-|
|![Screenshot_20230629_003543](https://github.com/Odongo006/Rails_blog_App/assets/25227022/dca2300f-25be-4fe0-b094-685e31d67394)|![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)|